src/garage/torch/modules/gaussian_mlp_module.py

```diff
@@ -266,9 +296,8 @@ def _get_mean_and_log_std(self, *inputs):
             *inputs: Input to the module.
 
         Returns:
-            tuple:
-                * mean (torch.Tensor): The mean of Gaussian distribution.
-                * std (torch.Tensor): The variance of Gaussian distribution.
+            torch.Tensor: The mean of Gaussian distribution.
+            torch.Tensor: The variance of Gaussian distribution.
```

Comment:
wait does it return the std, the log_std, or the variance? these distinctions matter.